### PR TITLE
Multiple changes

### PR DESCRIPTION
**Pull request by GTG3000**
- Added an "on_log_event" event to logger to allow for custom log entries by plugins
- Modified find_user util function to properly search for user id
- Adjusted editrole doc string to be more in line with reality
- Fixed !play not adjusting volume of the first song
- Made !musicban/!musicunban work again

### CODE REVIEW

File: plugins/logger.py [hunk 1]
```diff
@@ -12,7 +12,8 @@ class DiscordLogger(BasePlugin):
                 "message_delete",
                 "message_edit",
                 "member_join",
-                "member_remove"
+                "member_remove",
+                "log_event"
```
Comment (medeor413) [inline plugins/logger.py]: I would prefer that you add a separate event name for each event, for better control.

Comment (GTG3000) [inline plugins/logger.py]: The whole idea is for a cross-plugin support. Otherwise, it's going to require plugin-specific settings.

Comment (GTG3000) [inline plugins/logger.py]: redone

File: plugins/admin_commands.py [hunk 2]
```diff
@@ -25,25 +26,32 @@ class AdminCommands(BasePlugin):
...
-        if len(cnt) > 2:
-            searchstr = " ".join(cnt[2:])
-        else:
-            searchstr = ""
+        if len(args) > 2:
+            searchstr = args[2]
+        members = msg.mentions
+        if len(args) > 3:
+            for s in args[3:]:
+                if s.startswith("author:"):
```
Comment (medeor413) [inline plugins/admin_commands.py]: This isn't especially needed now that author is a separate author

**Pull request merged**